onnx/__init__.py

```diff
@@ -162,17 +164,43 @@ def _get_file_path(f: IO[bytes] | str | os.PathLike) -> str | None:
     return None
 
 
+def _get_serializer(
+    f: str | os.PathLike | IO[bytes] | None, fmt: _SupportedFormat | None
```

Comment:
Reordering the inputs would help align the order with their precedence.

Reply:
Do you mean it should be `_get_serializer(fmt, f)`? Just clarifying

Reply:
Done